Assets/Scripts/Environment/Hangar/Battery/BatteryWrapper.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class BatteryWrapper : MonoBehaviour
```

Comment:
I think it's worth adding ``RequireComponent()`` attributes for the Interactable and Charging.